Automattic/woocommerce-payments
# Add handling for refund failure in payment timeline

Fixes #1476 

#### Changes proposed in this Pull Request
- Add handling for `refund_failed` event in the payment timeline by adding a new line item.
- Update the payment summary status to reflect the refund failure.

Screenshot for reference with the updated timeline:

<img width="785" alt="Screenshot 2022-03-21 at 7 00 47 PM" src="https://user-images.githubusercontent.com/68396823/159271363-321bcbbc-cbc3-4af7-88ec-337bfc0074da.png">










#### Testing instructions





* Checkout this server PR: 1690-gh-Automattic/woocommerce-payments-server that introduces the new `refund_failed` timeline event.
* Follow the testing instructions there to get an event with refund failure.
* Verify that the new timeline event for refund failure is added.
* Verify that the status in the payment summary at the top is updated to `Refund failure`.

-------------------


**Post merge**
